test/ast-loader.js

```diff
@@ -0,0 +1,29 @@
+const postcss = require('postcss')
+const postcssPkg = require('postcss/package.json')
+const semver = require('semver')
+const loader = require('../src')
+
+const incomingVersion = semver.inc(postcssPkg.version, 'minor')
```

Comment:
You can take PostCSS version by `postcss().version` or by `result.processor.version` in PostCSS plugin API